xml/admin_ceph_upgrade.xml

```diff
@@ -325,6 +327,34 @@ mds standby for name = mds.456-another-mds
      &cephfs; load.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     MON and OSD nodes use the following settings:
+     <literal>osd_scrub_max_interval</literal> and
+     <literal>osd_deep_scrub_interval</literal>. OSDs use the settings to
+     determine when to run scrub, and MONs use the settings to check if they
+     need to show the warning.
+    </para>
+    <para>
+     In SES 5.5, MONs will not complain because by default the warnings are
+     disabled if the above settings are only in the <literal>[osd]</literal>
+     section.
+    </para>
+    <para>
+     In SES 6, the warnings are enabled. To ensure OSDs are deep-scrubbed on a
+     regular schedule and avoid warning messages during the upgrade to SES 6,
+     move all scrub-related settings to the <literal>[global]</literal>
+     section.
```

Comment:
I am not sure you get the point right. At least this is how I understand what I am reading here.

I would like it were something like below (please correct my English):

`osd_scrub_max_interval` and `osd_deep_scrub_interval` settings are used by both osd and mon daemons. The osd uses the settings to decide when to run scrub, and the monitor uses them to decide if a warning about scrub not running in-time (too long) should be show. So if non-default settings are used they should be visible by both osd and mon deamons (i.e. defined either in both [osd] and [mon] section, or in [global] section), otherwise the monitor may give a false alarm.

In SES 5.5 the monitor warning are disabled by default and the issue may be not noticed if the settings are overridden in [osd] section only, but when the monitors are upgraded to SES6 it will start to complain, because the warnings are enabled here by default. So if you have non-default scrub settings in your config, defined in [osd] section only, it is desirable to move them to [global] section before upgrade to SES6 to avoid false alarms about scrub not running in time.

Reply:
I think the wording above is good enough.